CNV/deploy-cnv.sh

```diff
@@ -95,5 +95,48 @@ done
 
 oc wait --for condition=ready pod -l app=kubevirt-node-labeller -n ${OPERATORS_NAMESPACE} --timeout=2400s
 
+echo "Setting up a bridge as the default interface"
+
+while [ "x" == "x$(oc get nodenetworkstate -o jsonpath='{.items[0].status.currentState.routes.running[?(@.destination=="0.0.0.0/0")].next-hop-interface}')" ]; do
```

Comment:
It would be better to check the destination to be machineCIDR, probably.

Reply:
Doesn't oc get -o jsonpath for a non-existing match give you a non zero return code?

If that were the case, you could simply have lines 100-104 be:
```
while ! default_iface=$(oc get nodenetworkstate -o jsonpath='{.items[0].status.currentState.routes.running[?(@.destination=="0.0.0.0/0")].next-hop-interface}'); do
    sleep 10
done```

Reply:
> It would be better to check the destination to be machineCIDR, probably.
Where can I get that value?

Reply:
I know it is in the install-config.yaml

Reply:
It's in install config, but I couldn't find at the moment whether it lives in other objects.

Reply:
I tried to find it in Kubernetes objects, but with no success. Will just read it from the config file.